### PR TITLE
Add remaining examples for express API reference

### LINKED ISSUE #1080: Add express examples for remaining express functions
## How To

Check off an example when you start working on it and push directly to this branch with each update.

* Examples are in `shiny/api-examples`
* Examples should be named `app-express.py` (or `app-{more-descriptive-name}-express.py`)
* If no example is needed for express use `@no_example("express")` before (below) `@add_example()` to suppress errors.
  ```python
  from ._docstring import add_example, no_example_express

  @add_example()
  @no_example("express")
  def thing_in_core():
      ...
  ```

## Todo

- [ ] Remove the early exit in `docs._renderer.check_if_missing_expected_example` so that it will error if a function is added to the Express docs that doesn't have an example.

## Remaining Apps to Translate

- [x] accordion
- [x] accordion_panel
- [x] as_fillable_container
- [x] as_fill_item
- [x] card
- [ ] ~card_body~
- [x] card_footer
- [x] card_header
- [x] close
- [x] data_frame
- [x] download
- [x] download_button
- [ ] ~download_link~
- [x] dynamic_route
- [x] event
- [x] extended_task
- [x] file_reader
- [x] include_css
- [x] include_js
- [x] insert_accordion_panel
- [x] insert_ui
- [x] modal
- [x] navset_hidden
- [x] nav_panel
- [x] notification_show
- [x] on_ended
- [x] on_flush
- [x] on_flushed
- [x] output_image
- [x] output_plot
- [x] output_table
- [x] output_text
- [ ] ~output_transformer~
- [x] output_ui
- [x] page_fixed
- [x] page_fluid
- [x] panel_title
- [x] poll
- [x] popover
- [x] remove_accordion_panel
- [x] remove_ui
- [x] Renderer
- [x] req
- [ ] ~row~
- [x] SafeException
- [x] send_custom_message
- [ ] ~showcase_bottom~
- [ ] ~showcase_left_center~
- [ ] ~showcase_top_right~
- [x] SilentCancelOutputException
- [x] SilentException
- [x] tooltip
- [x] update_accordion
- [x] update_accordion_panel
- [x] update_action_button
- [x] update_checkbox
- [x] update_checkbox_group
- [x] update_date
- [x] update_date_range
- [x] update_navs
- [x] update_numeric
- [x] update_popover
- [x] update_radio_buttons
- [x] update_select
- [x] update_selectize
- [x] update_sidebar
- [x] update_slider
- [x] update_text
- [x] update_tooltip
- [x] value_box

### PR DESCRIPTION
**Pull request by cpsievert**
Closes #1080 

@gadenbuie want to add the handful of `update_*()` examples you did?

### CODE REVIEW

**Comment by gadenbuie** [comment]
@cpsievert I added the remaining examples and took out the guardrails so that from now on missing examples in Express docs will throw in CI.

I also realized we hadn't accounted for Express functions being documented in the Core reference, so I updated the logic to automatically pick `app-express.py` for functions under `shiny/express`, even if we're in Core mode.

**Comment by cpsievert** [comment]
> I also realized we hadn't accounted for Express functions being documented in the Core reference

Oh, good catch, this is actually an oversight on my part...I think it'd be best not to include those Express functions in the Core docs?

File: docs/_renderer.py [hunk 1]
```diff
@@ -305,7 +306,7 @@ def is_no_ex_decorator(x):
         return
 
     # TODO: Remove shiny.express from no_req_examples when we have examples ready
```
Comment (gadenbuie) [inline docs/_renderer.py:308]:
```suggestion
```

File: shiny/api-examples/as_fillable_container/app-express.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+ui.markdown(
+    """\
+    # `as_fillable_container()`
+
+    For an item to fill its parent element,
+    * the item must have `as_fill_item()` be called on it
+    * the parent container must have `as_fillable_container()` called on it
+
+    Iff both methods are called, the inner child will naturally expand into its parent container.
```
Comment (gadenbuie) [inline shiny/api-examples/as_fillable_container/app-express.py:16]:
```suggestion
    If both methods are called, the inner child will naturally expand into its parent container.
```

File: shiny/api-examples/panel_title/rsconnect-python/panel_title.json
Comment (gadenbuie) [inline shiny/api-examples/panel_title/rsconnect-python/panel_title.json:1]: Was this committed by accident?

File: shiny/api-examples/panel_title/requirements.txt
Comment (gadenbuie) [inline shiny/api-examples/panel_title/requirements.txt:1]: This file might also not be needed.

File: shiny/api-examples/update_popover/app-express.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+with ui.popover(id="popover_id"):
+    ui.input_action_button("btn_w_popover", "A button w/ a popover", class_="mt-3")
+    "A message"
+
+
+@reactive.effect
+def _():
+    req(input.btn_show())
+
+    ui.update_popover("popover_id", show=True)
```
Comment (gadenbuie) [inline shiny/api-examples/update_popover/app-express.py:19]: I made these use `@reactive.event()` when I translated a similar app

```suggestion
@reactive.effect
@reactive.event(input.btn_show)
def _():
    ui.update_popover("popover_id", show=True)
```

File: shiny/api-examples/update_popover/app-express.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+
+@reactive.effect
+def _():
+    req(input.btn_close())
+
+    ui.update_popover("popover_id", show=False)
```
Comment (gadenbuie) [inline shiny/api-examples/update_popover/app-express.py:26]:
```suggestion
@reactive.effect
@reactive.event(input.btn_close)
def _():
    ui.update_popover("popover_id", show=False)
```

File: shiny/api-examples/update_popover/app-express.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+
+@reactive.effect
+def _():
+    req(input.btn_w_popover())
+    ui.notification_show("Button clicked!", duration=3, type="message")
```
Comment (gadenbuie) [inline shiny/api-examples/update_popover/app-express.py:32]:
```suggestion
@reactive.effect
@reactive.event(input.btn_w_popover)
def _():
    ui.notification_show("Button clicked!", duration=3, type="message")
```

File: shiny/api-examples/update_sidebar/app-express.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+ui.input_action_button("open_sidebar", label="Open sidebar", class_="me-3")
+ui.input_action_button("close_sidebar", label="Close sidebar", class_="me-3")
+
+
+@render.text
+def state():
+    return f"input.sidebar(): {input.sidebar()}"
+
+
+@reactive.Effect
```
Comment (gadenbuie) [inline shiny/api-examples/update_sidebar/app-express.py:16]:
```suggestion
@reactive.effect
```

File: shiny/api-examples/update_sidebar/app-express.py [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+@reactive.event(input.open_sidebar)
+def _():
+    ui.update_sidebar("sidebar", show=True)
+
+
+@reactive.Effect
```
Comment (gadenbuie) [inline shiny/api-examples/update_sidebar/app-express.py:22]:
```suggestion
@reactive.effect
```

**Review by gadenbuie — APPROVED** [review]
Looks good, just a few suggestions above, and maybe a couple of accidentally included files.

(I didn't read every line but I did skim all of the files.)

**Pull request merged**